src/njoy21/input/LEAPR/Card18/Skappa.hpp

```diff
@@ -19,15 +19,18 @@ struct Skappa {
       return false;
     } 
 
-    // Make sure skappa values are in increasing order
-    auto unsortedStart = std::is_sorted_until( skappas.begin(), skappas.end() );
-    if( unsortedStart != skappas.end() ){
-      auto dis = std::distance( skappas.begin(), unsortedStart );
-      Log::warning( "Kappa values are not in increasing order.");
-      Log::info( "kappa at index {} ({}) > kappa at index {} ({})",
-                   dis-1, unsortedStart[-1], dis, *(unsortedStart));
-      return false;
-    }
+//    While the manual says they need to be in increasing order, test problem
```

Comment:
I think this is one where we say that test 23 is not correct and needs to be changed. @whaeck what do you think?

Reply:
I sent an email to Ignacio to see what he says. He drafted the tests 22 and 23. If the requirement is to be dropped, we still need to change the user manual. I will update this when I get a reply.